### PR TITLE
feat: [EXT-3435]

### PR DESCRIPTION
**Pull request by Jwhiles**

### CODE REVIEW

File: .github/dependabot.yml [hunk 1]
```diff
@@ -12,3 +12,15 @@ updates:
     prefix: "fix"
     prefix-development: "chore"
     include: "scope"
+
+- package-ecosystem: npm
```
Comment (Jwhiles) [inline .github/dependabot.yml:16]: I think we will have to add each new template to dependabot individually, unless we add them as packages to the top level lerna config - or add another lerna config

File: .circleci/config.yml [hunk 2]
```diff
@@ -56,6 +56,9 @@ jobs:
       - run:
           name: Test
           command: npm run test
+      - run:
```
Comment (Jwhiles) [inline .circleci/config.yml:59]: As we add more templates, I think we can make a script here to loop through them and do this for each.

File: examples/templates/typescript-template/tsconfig.json [hunk 3]
```diff
@@ -0,0 +1,26 @@
+{
+  "compilerOptions": {
```
Comment (andipaetzold) [inline examples/templates/typescript-template/tsconfig.json:2]: Maybe we can use `@tsconfig/create-react-app` here, so we don't have to mess around with the config ourselves.

https://www.npmjs.com/package/@tsconfig/create-react-app

But as this is just a copy, feel free to ignore this comment

**Review by andipaetzold — COMMENTED** [review]
I assume this is just a copy from create-contentful-app, so nothing to worry from the implementation side.

I am just not sure in which folder the templates should live. Nesting under `examples` feels a little strange. Also having an additional `[...]-template` in the folder name seems verbose.
I'd rather put it directly `/examples/typescript-template` or `/templates/typescript`. The latter would be less verbose when specifying the template in CCA.

**Comment by Jwhiles** [comment]
> I assume this is just a copy from create-contentful-app, so nothing to worry from the implementation side.
> 
> I am just not sure in which folder the templates should live. Nesting under `examples` feels a little strange. Also having an additional `[...]-template` in the folder name seems verbose. I'd rather put it directly `/examples/typescript-template` or `/templates/typescript`. The latter would be less verbose when specifying the template in CCA.
> 
> wdyt?

I agree about the path, I read in the acceptance criteria that they should be in the examples folder, but let's move to just /templates

File: .circleci/config.yml [hunk 4]
```diff
@@ -57,8 +57,8 @@ jobs:
           name: Test
           command: npm run test
       - run:
-          name: Build template-typescript
-          command: cd examples/templates/typescript-template && npm i && npm run build
+          name: Build TypeScript template
+          command: cd templates/typescript && npm i && npm run build
```
Comment (andipaetzold) [inline .circleci/config.yml:61]: Should we also run the tests here? If not, you can merge.

**Review by andipaetzold — APPROVED** [review]

**Pull request merged**